mosaic-runtime/src/main/kotlin/com/jakewharton/mosaic/output.kt

```diff
@@ -47,13 +47,19 @@ internal object AnsiOutput : Output {
 			}
 
 			// If the new output contains fewer lines than the last output, clear those old lines.
-			for (i in 0 until lastHeight - lines.size) {
+			val extraLines = lastHeight - lines.size
+			for (i in 0 until extraLines) {
 				if (i > 0) {
 					append('\n')
 				}
 				append("\u001B[K") // Clear line.
 			}
 
+			// Move cursor back up to end of the new output
```

Comment:
```suggestion
			// Move cursor back up to end of the new output.
```
nit. we follow https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html